src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -702,6 +702,92 @@ class FlatbuffersExpressionEvaluator {
       }
     }
 
+    case tuix::ExprUnion_In:
+    {
+      auto c = static_cast<const tuix::In *>(expr->expr());
+      size_t num_children = c->children()->size(); 
+      bool result = false;
+
+      if (num_children < 2){
```

Comment:
Can you verify what kind of types `IN` handles? What if the user passes in a CalendarType?

Reply:
Used a dynamic typed comparison method that should work with different types